### PR TITLE
doc: describe number-of-shards config, #26920

### PR DESCRIPTION
**Pull request by patriknw**
Refs #26920

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/cluster-sharding.md [hunk 1]
```diff
@@ -122,16 +122,34 @@ is used but `tell` or any of the other @ref:[Interaction Patterns](interaction-p
...
+function of the `entityId`. For a specific entity identifier the shard identifier must always
+be the same. Otherwise the entity actor might accidentally be started in several places at the same time.
+
+By default the shard identifier is the absolute value of the `hashCode` of the entity identifier modulo
+the total number of shards. The number of shards is configured by:
+
+@@snip [reference.conf](/akka-cluster-sharding-typed/src/main/resources/reference.conf) { #number-of-shards }
+
+As a rule of thumb, the number of shards should be a factor ten greater than the planned maximum number of
+cluster nodes. It doesn't have to be exact. Less shards than number of nodes will result in that some nodes will
```
Comment (chbatey) [inline akka-docs/src/main/paradox/typed/cluster-sharding.md]: s/Less shards/Fewer shards/

**Review by chbatey — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/typed/cluster-sharding.md (hunk 1 above)
Comment (patriknw) [reply akka-docs/src/main/paradox/typed/cluster-sharding.md]: fixed

**Pull request merged**